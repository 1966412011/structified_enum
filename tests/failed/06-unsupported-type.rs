use structified_enum::structify;

#[structify]
struct Foo;

fn main() {}