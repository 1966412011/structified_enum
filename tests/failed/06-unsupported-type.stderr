error: expected `enum`
 --> tests/failed/06-unsupported-type.rs:4:1
  |
4 | struct Foo;
  | ^^^^^^
